README.md

```diff
@@ -35,11 +35,22 @@ To install from source:
 make install
 ```
 
-## DigitalOcean API
+## **`pydo` Quickstart**
 
-To support all of DigitalOcean's HTTP APIs, a generated library is available which will expose all the endpoints:  [pydo](https://github.com/digitalocean/pydo/tree/main/src/digitalocean).
+> A quick guide to getting started with the client.
 
-Find below a working example for GET a ssh_key ([per this http request](https://docs.digitalocean.com/reference/api/api-reference/#operation/sshKeys_list)) and printing the ID associated with the ssh key. If you'd like to try out this quick example, you can follow [these instructions](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/) to add ssh keys to your DO account.
+`pydo` must be initialized with `pydo.Client()`. A DigitalOcean API Token is required. The token can be passed explicitly to `pydo.Client()`, as such:
+
+```python
+import os
+from pydo import Client
+
+client = Client(token=os.getenv("$DIGITALOCEAN_TOKEN"))
```

Comment:
`getenv` takes the environment variable name as parameter, without `$` from shell syntax